### ISSUE TITLE
Copy-book resolution error 

### ISSUE THREAD

**Issue by Reinhard-Prehofer** [uwol/cobol85parser#45]
Suppose you include a copy book in speficying an extension in the name of the copybook but using no qoutes. Thus the Copy-statement looks like:

`  COPY abc.cpy`

=> the copy book is resolved and found correctly, but the ".cpy" extension is split into a new line, somehow looking like:
``` (here are the lines of the copy book ...)
      *>  ****************************************************************
CPY.
(Next statement after the copy abc.cpy statement ...)
```

A very close look shows, that the CPY. starts in column 1 and thus should not be interpreted when also having specified the FIXED format option ... 
       
And this leads to a parsing error as follows:
Parsing file ab2600145.CBL.
line 171:0 mismatched input 'CPY' expecting <EOF>
Collecting units in file ab2600145.CBL.

Well - I know, it is negotiable if this really is a parsing error, 
of course a 
COPY abc
or a 
COPY 'abc.cpy'
work correctly.
the compilers we are using here on site, also resolve a copy abc.cpy Statement correctly => so personally I tend to say, yes it is an error.

**Comment by uwol** [comment]
Yes, this definitely is an error, as the preprocessor cannot parse successfully and only by pure luck is able to identify the copy book :-).

Since #43 this also should lead to an exception anyway, so I have to fix this.

**Comment by uwol** [comment]
Fixed in 4393a470edef8f33db4043c02166d3c5e5402320. Added a [unit test](https://github.com/uwol/cobol85parser/blob/4393a470edef8f33db4043c02166d3c5e5402320/src/test/resources/io/proleap/cobol/preprocessor/copy/cobolword/variable/CopyCblWord.cbl#L5).

Thanks!
